binder/environment.yml

```diff
@@ -1,33 +1,40 @@
+# minimal environment to install hydromt and run examples
 name: hydromt
 
 channels:
   - conda-forge
 
 dependencies:
   - affine
-  - click
   - black
   - bottleneck
   - cartopy # to run examples
+  - click
   - dask
   - descartes # to run examples
   - entrypoints
   - flit>=3.2
   - gdal>=3.1
-  - geopandas
+  - geopandas>=0.10
   - jupyter # to run examples
   - netcdf4
   - matplotlib # to run examples
   - numba
   - numpy
   - openpyxl
   - pandas
+  - pcraster # optional
   - pip
-  - pyflwdir>=0.5.0
+  - pyflwdir>=0.5.4
   - pygeos>=0.8
-  - python>=3.8
+  - python>=3.7
   - rasterio
   - requests
+  - rioxarray
   - scipy
+  - toml
   - xarray
-  - zarr
\ No newline at end of file
+  - zarr
+  - pip:
+    - hydroengine
```

Comment:
I see you added hydroengine, do we need it in hydromt core? I thought just hydromt_wflow

Reply:
Removed hydroengine here and marked it optional in hydromt-dev.